object_tracker/object_tracker.py

```diff
@@ -33,7 +34,7 @@ def run_frame(self, img):
                 flag = "normal"
                 score = 1
             else:
-                print("failed to match features")
+                logging.error("failed to match features")
         if (
             self._tracker_counter % self._tracker_run_iter == 0
```

Comment:
I would make it more robust to a case where self._tracker_run_iter is set to 0 (and 1). Currently it will crash.